src/task/style.css

```diff
@@ -258,3 +258,10 @@ footer {
   color: rgb(255, 92, 92);
   font-size: 0.75rem;
 }
+
+#assignee {
+  background-repeat: no-repeat;
+  background-position: right;
+  background-size: 42px;
```

Comment:
Can we use the shorthand [background](https://developer.mozilla.org/en-US/docs/Web/CSS/background) CSS property for all these?